resources/shaders/Toon.frag

```diff
@@ -40,14 +39,15 @@ layout (std140, set=0, binding=1) readonly buffer PointLightsArray
     PointLight lights[];
 } pointLights;
 
-layout (set = 1, binding = 2) uniform sampler2D textures[];
-layout (set = 1, binding = 4) uniform samplerCube cubeMaps[];
 layout (set = 0, binding = 5) uniform EnvironmentDataBuffer
 {
+    mat4 cameraViewProjection;
     vec4 cameraWorldPosition;
     int skyboxCubemapIndex;
 } environmentData;
 
+layout (set = 1, binding = 2) uniform sampler2D textures[];
+layout (set = 1, binding = 4) uniform samplerCube cubeMaps[];
 layout (location = 0) in vec3 inFragPosition;
```

Comment:
I like having a newline between the descriptor resources and the resources that come from the vertex shader.